content-src/reducers/reducers.js

```diff
@@ -21,8 +21,34 @@ function setRowsOrError(type) {
   };
 }
 
+function setSearchState(type) {
```

Comment:
Since this reducer is unique to `Search` you can just have this be a single function